kafka_consumer/tests/conftest.py

```diff
@@ -54,14 +67,19 @@ def dd_environment(mock_local_kafka_hosts_dns):
     with docker_run(
         DOCKER_IMAGE_PATH,
         conditions=[
-            WaitFor(create_topics, attempts=60, wait=3),
-            WaitFor(initialize_topics),
+            WaitFor(grant_acl_permissions),
+            # TODO: This create_topics still does not work yet
+            WaitFor(create_topics, attempts=2, wait=2),
```

Comment:
I changed the `attempts` and `wait` to 2 just to make the error logs easier to read